src/braket/pennylane_plugin/braket_device.py

```diff
@@ -181,10 +181,6 @@ def apply(
 
         # Add operations to Braket Circuit object
         for operation in operations + rotations:
-            if operation.base_name not in supported_operations():
-                raise NotImplementedError(
-                    f"Braket PennyLane plugin does not support operation {operation.name}."
-                )
```

Comment:
I added this since one of the unit tests was failing since the Mock op didn't have any parameters. I noticed that you added that in another code change this commit, so it's nice we can remove this piece